src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -148,7 +148,11 @@ function assertValidProps(component, props) {
     props.style == null || typeof props.style === 'object',
     'The `style` prop expects a mapping from style properties to values, ' +
     'not a string. For example, style={{marginRight: spacing + \'em\'}} when ' +
-    'using JSX.'
+    'using JSX.%s',
+    component._currentElement._owner ?
+      ' Check the render method of ' +
+      component._currentElement._owner.getName() + '.' :
+        ''
```

Comment:
Formatting feels off here. Can we assign `owner` before the `invariant` call and use that (can also use a template string instead of concatting the `getName()`, it'll fit on 1 line when we have `owner`)